nprogress.js

```diff
@@ -221,7 +221,28 @@
    */
 
   NProgress.getElement = function() {
-    return document.querySelector(Settings.parent + ' > .nprogress');
+    var parent = NProgress.getParent();
+    // if the parent contains nested NProgress instances, select just the direct child
+    if (parent) {
+      return Array.prototype.slice.call(parent.querySelectorAll('.nprogress')).find(p => p.parentElement === parent);
```

Comment:
I see that you used the method find of arrays which is not supported on internet explorer.
Sorry it is been long time but as far as i remember the library should work on all browsers. So i don't want to introduce breaking change. You can use .filter and take the first element or to be more efficient have a for loop.

Reply:
Sounds good! Working on that now.

Reply:
Made the change. Unit tests are passing, testing it in my app now.

Reply:
Confirmed working both in my app and in the tests.